docs/data/date-pickers/time-picker/TimePickerOpenTo.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { DemoContainer } from 'docsx/src/modules/components/DemoContainer';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { Unstable_MobileNextTimePicker as MobileNextTimePicker } from '@mui/x-date-pickers/MobileNextTimePicker';
+
+export default function TimePickerOpenTo() {
+  return (
+    <LocalizationProvider dateAdapter={AdapterDayjs}>
+      <DemoContainer>
+        <MobileNextTimePicker label={'"minutes"'} openTo="minutes" />
+        <MobileNextTimePicker
+          label={'"seconds"'}
+          openTo="seconds"
+          views={['minutes', 'seconds']}
```

Comment:
This example is somewhat clumsy, especially after value is selected 🤔 
Maybe we could specify format to avoid such confusion?
```suggestion
          views={['minutes', 'seconds']}
          format="HH:mm:ss A"
```

Reply:
With LTS my mentioned clumsiness is still visible 🤔 
Hours are shown and set as `12`. 🤷 
Or am I missing something?

https://user-images.githubusercontent.com/4941090/210554585-fc38c98b-4d91-45fb-9025-f2e5f356deec.mov

Reply:
Nevermind, that was my mistake using `HH` (24h) instead of `hh` (12h). 🙈 
It could still use some improvement at some point as this causes a disconnect between actual format and toolbar format, but I'm fine with the solution now. 👌

Reply:
In #7380 I am hiding the `AM / PM` switch which does not make a lot of sense if their is no hour view.
I can set a format without the hour here by the way, that would be closer from the views.

Reply:
Yeah, fair point. AM/PM is also super confusing in such case...
Maybe just setting `format="mm:ss"` would be the most fool-proof way for this demo. 🤔

Reply:
For the toolbar, at some point we could imagine doing the same parsing as in the field and generate the UI based on it.

Reply:
The two demos should be coherent now